msbuild/domoticz.vcxproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{6AD96441-5B2C-467B-AAB6-A1B8368D503A}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>domoticz</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
Please undo this change

Reply:
Can I update once than add it to .gitignore - so it will not be updating any longer?
Without this version and libraries it will not compile and run on VS2017 (I use most recent VS and Windows SDK)

Reply:
Unfortunately with gitignore it will ignore the file completely.
But you can modify the file locally, only when you commit your changes, make sure not to commit this file.
As you are using VS2017 (as do i), you can use the 'vstudio tortoisegit toolbar' or use tortuisegit